dashboard/routings/result.py

```diff
@@ -197,7 +198,8 @@ def get_hits_per_hour(end):
 def get_time_per_version_per_user(end, versions):
     user_data = {}
     data = [t.execution_time for t in get_all_measurement(end)]
-    average = sum(data) / len(data)
+    # compute the average for determining the default size
+    average = math.sqrt(sum(data) / len(data))/1250
```

Comment:
magic constant!

Reply:
It is now defined as a constant at the top of the file, is this what you had in mind?